covid/tests.py

```diff
@@ -22,6 +22,8 @@ def test_process(self):
         # the last entry in the data is the day before `run_date`!
         assert df_processed.xs('NY').index[-1] < run_date
         assert df_processed.xs('NY').index[-1] == (run_date - pandas.DateOffset(1))
+        assert "positive" in df_processed.columns
+        assert "total" in df_processed.columns
 
 
 class TestGenerative:
```

Comment:
These tests are comprehensive in checking model structure is right. Would suggest for future effort including a test that fitting some "very obvious data" returns the "very obvious" results to check that model computation is probably correct

Reply:
in general yes, but the model takes a long time to sample and we don't have a generated test dataset yet...